cogs/utility.py

```diff
@@ -59,6 +61,21 @@ async def serverinfo(self, ctx):
         embed.add_field(name="Features", value=features)
         await ctx.respond(embed=embed)
 
+    @slash_command(brief="Get all server stickers & emojis!")
+    @discord.default_permissions(manage_guild=True)
+    @commands.cooldown(1, 600, commands.BucketType.user)
+    async def emoji_downloader(self, ctx):
+        await ctx.defer()
+        with zipfile.ZipFile('emoji_and_stickers.zip', 'w') as zipped_f:
+            for emoji in ctx.guild.emojis:
+                zipped_f.writestr(emoji.name + emoji.url[-4:], await emoji.read())
```

Comment:
I'd like a delay between each emote download.

Reply:
Do you reckon half a second is enough? Else it might take a long time for the bot respond

Reply:
One emote per second.